src/lib.rs

```diff
@@ -207,6 +207,8 @@ impl fmt::Display for Ratio {
 pub enum Colorspace {
```

Comment:
If we make a breaking change then would it help if well mark this `non_exhaustive`?

[libavm](https://aomedia.googlesource.com/avm/+/refs/heads/109-encoder-crash-with-debug-build/common/y4menc.c) encoding has a few additional color spaces (even though I can't seem to locate where those are decoded, the string sets are different there).

Reply:
Good idea. done.